### PR TITLE
BTA-8016 Update SandBox Paying Documentation

### PR DESCRIPTION
**Pull request by thirunjuguna**
BTA-8016

This PR Adds documentation for Sandbox Payin Endpoint 

Screenshot 2022-03-14 at 22 18 09

Screenshot 2022-03-14 at 22 02 13

### CODE REVIEW

File: _docs/sandbox-testing.md [hunk 1]
```diff
@@ -14,6 +14,122 @@ To facilitate testing our sandbox environment has a few features that can help i
...
+  end
+  raise e
+end
+```
+{% endcapture %}
+{% include language-tabbar.html prefix="gen-payin-transaction-call" raw=data-raw csharp=data-csharp vb=data-vb java=data-java js=data-js php=data-php ruby=data-ruby %}
+
 # Forcing collections to succeed
 
 When using our [collections feature]({{ "/docs/additional-features/" | prepend: site.baseurl }}#collections-from-senders) you can force the collection to succeed. Please see the documentation of this feature in our [API specification guide](https://api.transferzero.com/documentation/#/Transactions/payin-transaction)
```
Comment (archmagos) [inline _docs/sandbox-testing.md:135]: Should we also remove this, or incorporate it into the additions you've made?

Comment (archmagos) [inline _docs/sandbox-testing.md:135]: Should we also remove this, or incorporate it into the additions you've made?

Comment (thirunjuguna) [reply _docs/sandbox-testing.md]: Thanks

File: _docs/sandbox-testing.md [hunk 2]
```diff
@@ -14,9 +14,121 @@ To facilitate testing our sandbox environment has a few features that can help i
 
 You can use the `POST /transactions/[TRANSFERZERO_TRANSACTION_ID]/payout` endpoint with an empty body to force a funded, but not yet paid transaction to succeed. You can use this endpoint to test out if you are receiving successful transaction webhooks properly.
 
-# Forcing collections to succeed
+# Forcing transactions to pay in
 
-When using our [collections feature]({{ "/docs/additional-features/" | prepend: site.baseurl }}#collections-from-senders) you can force the collection to succeed. Please see the documentation of this feature in our [API specification guide](https://api.transferzero.com/documentation/#/Transactions/payin-transaction)
+You can use the `POST /transactions/[TRANSFERZERO_TRANSACTION_ID]/payin` endpoint with a `PayinMethodRequest` body to force a successful payin. You can use this endpoint to test out if you are receiving successful transaction webhooks properly.See the examples below using our SDKs.
```
Comment (archmagos) [inline _docs/sandbox-testing.md:19]:
```suggestion
You can use the `POST /transactions/[TRANSFERZERO_TRANSACTION_ID]/payin` endpoint with a `PayinMethodRequest` body to force a successful payin, for example to test out if you are receiving successful transaction webhooks properly. See the examples below using our SDKs.
```
What do you think to this?

File: _docs/sandbox-testing.md [hunk 3]
```diff
@@ -14,9 +14,121 @@ To facilitate testing our sandbox environment has a few features that can help i
...
+POST /v1/transactions/e77f89fa-8371-496c-ae73-e56697fc08d8/payin
+{
+  "payin_method": {
+    "type": "GHS::Mobile"
+  }
+}
+```
+{% endcapture %}
+
+{% capture data-csharp %}
```
Comment (sztupy) [inline _docs/sandbox-testing.md:32]: Were these generated using the codegenerator?

Comment (thirunjuguna) [reply _docs/sandbox-testing.md:53]: @sztupy  this has been updated

**Review by sztupy — APPROVED** [review]

**Pull request merged**